client/rest/src/main/java/org/elasticsearch/client/RestClient.java

```diff
@@ -91,231 +87,138 @@
  * <p>
  * Requests can be traced by enabling trace logging for "tracer". The trace logger outputs requests and responses in curl format.
  */
-public class RestClient implements Closeable {
-
+public class RestClient extends AbstractRestClientActions implements Closeable {
     private static final Log logger = LogFactory.getLog(RestClient.class);
 
+    /**
+     * The maximum number of attempts that {@link #nextNode(NodeSelector)} makes
+     * before giving up and failing the request.
+     */
+    private static final int MAX_NEXT_NODES_ATTEMPTS = 10;
```

Comment:
This is no longer used, could be removed

Reply:
++